### PR TITLE
Improve Logo

### PR DESCRIPTION
**Pull request by akaszynski**
## Better Logo

This is one of those relatively minor things that ends up taking forever to do right.

Our logo doesn't include the "Py", which seems a bit confusing, and is partially why our logo isn't that promoted.  This PR adds the "Py", and makes it possible to generate it programmatically as a demo with:

```py

from pyvista import demos

demos.plot_logo()

```
#### New Logo
pyvista_logo

#### Old Logo
pyvista_logo_old

@pyvista/developers, please let me know what you'd change to make the logo better.  I'm not a graphics designer, so multiple opinions would be nice here.

### CODE REVIEW

**Comment by GuillaumeFavelier** [comment]
I'm +100 to include the "Py" and have a function to generate it, so for me it's already a big step forward :+1:

**Comment by adeak** [comment]
Very nice initiative!

I'm terrible at design. So just one remark: I like how in the old logo every letter has a very different representation. Can we maybe do something to distinguish the P and y from each other?

**Comment by akaszynski** [comment]
> I'm +100 to include the "Py" and have a function to generate it, so for me it's already a big step forward

The intent was to have "py" look plain and use the Python colors to denote that we're extending Python.  I also wanted to avoid it looking too "busy".

**Review by banesullivan — APPROVED** [review]
Love it!

File: pyvista/demos/logo.py [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+Logos generated with:
+plot_logo(screenshot='/home/alex/python/pyvista/docs/_static/pyvista_logo.png', window_size=(1920, 1080))
+plot_logo(screenshot='/home/alex/python/pyvista/docs/_static/pyvista_logo_sm.png', window_size=(960, 400), off_screen=True)
+plot_logo(screenshot='/home/alex/python/pyvista/docs/_static/pyvista_logo_sm_sq.png', window_size=(960, 960), off_screen=True)
+
+cpos = [(-0.3654543687422538, 1.1098808905156292, 9.073223697728247),
+        (2.553950615449191, 0.34145688392081264, 0.06127122762851659),
+        (0.019308531920309947, 0.996708840795678, -0.07873161547192065)]
+
+plot_logo(screenshot='/home/alex/python/pyvista/docs/_static/pyvista_logo_sm_sq.png', window_size=(960, 960), cpos=cpos, off_screen=True)
```
[the remark this answers is not in the document]
Comment (akaszynski) [reply pyvista/demos/logo.py]: Resolved in https://github.com/pyvista/pyvista/pull/1236/commits/0ecf8948332f20f3f5ee92bc7d787454e5c8ac01

File: pyvista/demos/demos.py [hunk 2]
```diff
@@ -5,7 +5,7 @@
 
 
 def glyphs(grid_sz=3, **kwargs):
-    """Plot several parametric supertoroids using VTK's glyph table functionality."""
+    """Create several parametric supertoroids using VTK's glyph table functionality."""
     n = 10
     values = np.arange(n)  # values for scalars to look up glyphs by
```
Comment (adeak) [inline pyvista/demos/demos.py:10]: We don't even need to pass this explicitly anymore, and the glyph table option will generate the same `values`. We might still keep it for educational purposes, perhaps?

Comment (akaszynski) [reply pyvista/demos/demos.py:10]: I'll be using this within another project/branch, which is partially why I updated the logo anyway.

**Comment by tkoyama010** [comment]
I'm a fan of this logo.

**Pull request merged**